articles/security/store-tokens.md

```diff
@@ -47,6 +47,18 @@ There are different options to control the lifetime of a cookie:
 * Cookies can be persistent (not destroyed after the browser is closed) with an expiration.
 * Cookies can be read by both the JavaScript and the server side code or only server side if the `httpOnly` flag is set.
 
+## Understanding Sessions and Cookies
+
+:::warning
+Please be sure to refer to the [Lock API documentation](/libraries/lock/v10/api) for the most up-to-date code snippets.
+:::
+
+<%= include('../../_video', { id: 'paxlbixuya' }) %>
+
+This video will show you how to handle session data when building a web app. It will help you understand how your application uses cookies and sessions to manage the state of an authenticated user. This video example uses Node.js with Passport, but the techniques apply to any traditional server-based web application.
+
+<div style="height: 50px"></div>
```

Comment:
There's an unusual amount of whitespace after this text, why is that? See https://auth0-docs-content-pr-6124.herokuapp.com/docs/security/store-tokens#understanding-sessions-and-cookies